### PR TITLE
[Merged by Bors] - TY-2531 Tool to create mock model

### PR DESCRIPTION
**Pull request by biernacki**
[TY-2531 - Tool to create mock model](https://xainag.atlassian.net/browse/TY-2531)
[TY-2538 - Change test to use mock models](https://xainag.atlassian.net/browse/TY-2538)

This PR contains a Python script that generates dummy `bert` or `smbert` onnx models which have the same inputs and outputs as the models we are using in production.

I've generated these two models using that script and added them to version control in the assets folder of our flutter example. Those assets are then used by our integration tests. I've adjusted a helper method which prepares the asset `Manifest` so that it contains mocked versions of the models instead of real ones.

### CODE REVIEW

File: onnx-tool/README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+## Mock (sm)bert model generator
+
+Simple python script to generate dummy bert (kpe) and smbert ONNX models. It creates graph definition with the same inputs and outputs as original v0001 models.
+
+# 📌 Prerequisites
+You need to have [Python](https://www.python.org/) installed.
+
+# 🏗 Usage
+
+First install needed dependencies and then run the script specifiying the type of the model (`bert` or `smbert`) and output path. By default the output path is `"."` (the current directory).
```
Comment (Robert-Steiner) [inline onnx-tool/README.md:10]:
```suggestion
First install needed dependencies and then run the script specifying the type of the model (`bert` or `smbert`) and output path. By default the output path is `"."` (the current directory).
```

File: onnx-tool/src/mock_onnx_tool.py [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+        graph_def,
+        producer_name='ai.onnx',
+        opset_imports=[helper.make_opsetid('', 12)],
+    )
+
+    onnx.checker.check_model(model_def)
+    print(f"The model is checked: \N{heavy check mark}")
+
+    onnx.save(model_def, model_path)
+    print(f"The model is saved unde {model_path}: \N{heavy check mark}")
```
Comment (Robert-Steiner) [inline onnx-tool/src/mock_onnx_tool.py:21]:
```suggestion
    print(f"The model is saved under {model_path}: \N{heavy check mark}")
```

**Review by Robert-Steiner — COMMENTED** [review]
Looks good to me!

**Review by Robert-Steiner — APPROVED** [review]
🚀

File: onnx-tool/src/mock_onnx_tool.py [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+    )
+
+    onnx.checker.check_model(model_def)
+    print(f"The model is checked: \N{heavy check mark}")
+
+    onnx.save(model_def, model_path)
+    print(f"The model is saved under {model_path}: \N{heavy check mark}")
+
+def create_smbert_graph() -> GraphProto:
+    # Create one input (ValueInfoProto)
```
[the remark this answers is not in the document]
Comment (biernacki) [reply onnx-tool/src/mock_onnx_tool.py]: just a typo I guess, fixed it

**Comment by biernacki** [comment]
bors merge

**Pull request closed without merging**